app/assets/stylesheets/_base.scss

```diff
@@ -1529,7 +1539,7 @@ body {
     padding: 16px;
     margin-bottom: 40px;
     margin-top: 14px;
-    max-height: 378px;
+    max-height: 750px;
```

Comment:
To accommodate a long list of services. Hopefully, it works for most or even all cases.